README.md

```diff
@@ -245,11 +275,11 @@ use it and you will have a positive impact on the world.
 
 Documentation has two main components. The first component is help files, which go into the `man/` folder. The second 
 component is vignettes which will go in a folder called `vignettes/ which you will have to create. I'll tackle 
-each of these separately. 
+each of these separately. As a rule, the user should be able to use the package reading only the MAN files, and the vignettes
+may be a complete or summary of what you can do with the package.
```

Comment:
A vignette is a crucial component of a Bioconductor package. That fact should not be understated.